act-rules-format.bs

```diff
@@ -242,7 +251,7 @@ An [=atomic rule=] MUST contain one or more expectations. An expectation is an a
 When all expectations are true for a test target, the test target `passed` the rule. If one or more expectations are false, the test target `failed` the rule. When atomic rules are used in composite rules, the outcome of the composite rules can be `passed` when the outcome of individual atomic rule is `failed`, depending on the [expectations](#expectations-composite) of the composite rule.
 
 <div class=example>
-  <p>A rule for labels of HTML `input` elements might have the following expectations:</p>
+  <p><strong>Example</strong>: A rule for labels of HTML `input` elements might have the following expectations:</p>
```

Comment:
Between this change and the previous change on line 227, I'm getting confused about the use of "applicability' and 'expectations' as the listed statements used as examples of each seem like the exact same kind of check.